### PR TITLE
Add random stuff

### PR DESCRIPTION
**Pull request by garyluu**
Add command for documentation.  Creates docs in the `docs` folder.  In the future github pages can be pointed to the docs folder of the master branch (Travis can auto-generate docs on master branch).

Also try to remove some snyk warnings (bootstrap lied, there's a newer non-breaking version)

Another shot at resetDb

### CODE REVIEW

**Comment by garyluu** [comment]
forgot to skip ci

File: scripts/resetDb.sql [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SELECT pg_cancel_backend(pid) FROM pg_stat_activity WHERE pid IN (SELECT pid FROM pg_locks) AND query NOT LIKE '%pg_stat_activity%';
```
Comment (denis-yuen) [inline scripts/resetDb.sql]: document this (in this file)

**Comment by garyluu** [comment]
Fixes a few more cypress tests.  The postgres fix didn't work, so now trying to rule out the possibility it's just slow.

**Comment by garyluu** [comment]
Bootstrap issue is https://github.com/ga4gh/dockstore/issues/1899  . There's a newer version of bootstrap since then (without breaking changes).  Will see if that fixes it.

**Comment by garyluu** [comment]
I forgot to skip ci again.

**Pull request merged**